diamond-api/src/find.rs

```diff
@@ -157,7 +158,7 @@ mod tests {
     use super::*;
 
     #[test]
-    fn url_serialize() -> Result<(), failure::Error> {
+    fn url_serialize() -> Result<(), serde_urlencoded::ser::Error> {
```

Comment:
Emiting `Box<dyn std::error::Error>` in tests is just fine. No need to be such specific :)